web/themes/custom/pineapple/sass/field/_field-label.scss

```diff
@@ -0,0 +1,17 @@
+.inline-field-label {
+  color: $primary;
+  padding: $unit;
+  margin-right: $unit;
+  display: inline;
+
+  .field__label,
+  .field__value {
+    display: inline;
+    text-transform: uppercase;
+  }
+
+  .field__label {
+    padding-right: 0;
+    font-weight: bolder;
+  }
+}
```

Comment:
The field folder should probably been in components please read drupal theming standards docs: https://www.drupal.org/docs/develop/standards/css/css-file-organization-for-drupal-8#file-structure https://www.drupal.org/node/1887918#separate-concerns

Reply:
Done. Thank you.